ts/Core/Axis/AxisDefaults.ts

```diff
@@ -1278,6 +1278,17 @@ namespace AxisDefaults {
          * @apioption xAxis.minorTicks
          */
 
+        /**
+         * The number of minor ticks per major tick. Works for `linear`,
+         * `logarithmic` and `datetime` axes.
+         *
+         * @sample {highcharts} highcharts/xaxis/minortickspermajor/
```

Comment:
The sample path says `xaxis` while the demo actually shows `yaxis`. Better to rename the sample.